medical-system/src/hooks/usePatients.tsx

```diff
@@ -0,0 +1,55 @@
+import {
+  addDoc,
+  collection,
+  deleteDoc,
+  doc,
+  updateDoc,
+} from "firebase/firestore";
+import { db } from "../auth/firebase-config";
+import useSWRMutation from "swr/mutation";
+import { User } from "../components/shared/Table";
+
+export function usePatients() {
```

Comment:
może bardziej usePatientsActions albo coś tego typu? żeby było bardziej wiadomo że to jest hook który zwraca dostepne akcje per pacjent